web/new_alt_text_manager.js

```diff
@@ -128,16 +130,16 @@ class NewAltTextManager {
     textarea.addEventListener("focus", () => {
       this.#wasAILoading = this.#isAILoading;
       this.#toggleLoading(false);
+      this.#toggleTitleAndDisclaimer();
     });
     textarea.addEventListener("blur", () => {
-      if (textarea.value) {
-        return;
+      this.#toggleTitleAndDisclaimer();
+      if (!textarea.value) {
+        this.#toggleLoading(this.#wasAILoading);
       }
```

Comment:
Nit: Elsewhere these methods (always) seem to be called in the "other" order, so can we perhaps do that here as well for consistency?
```suggestion
      if (!textarea.value) {
        this.#toggleLoading(this.#wasAILoading);
      }
      this.#toggleTitleAndDisclaimer();
```